### DISCUSSION TITLE
Is the RNAlysis package compatible with macOS running on M series CPUs?

### DISCUSSION THREAD

**Discussion by santataRU** [GuyTeichman/RNAlysis#37] [General]
Hi all,

Is the RNAlysis package compatible with macOS running on M series CPUs?

Thanks,

Xiao

**Comment by GuyTeichman** [comment]
Hi Xiao,
I don't own an M series MacBook, but it seems to run well on M series GitHub runners, so I'm quite certain it will work for you too.

Please let me know if you run into any issues.

Best,
Guy.
